Resources/Prototypes/Reagents/Consumable/Drink/drinks.yml

```diff
@@ -317,3 +317,12 @@
   physicalDesc: strong-smelling
   color: "#485000"
   spritePath: rewriter.rsi
+
+- type: reagent
+  id: Kvass
+  name: reagent-name-kvass
+  parent: BaseDrink
```

Comment:
Is Kvass not slightly alcoholic?

Reply:
In such barrels, kvass has a very very small percentage of alcohol that it is more like soda